akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphBroadcastSpec.scala

```diff
@@ -214,6 +216,25 @@ class GraphBroadcastSpec extends AkkaSpec {
       upsub.expectCancellation()
     }
 
+    "alsoTo must broadcast" in assertAllStagesStopped {
+      val p, p2 = TestSink.probe[Int](system)
+      val (ps1, ps2) = Source(1 to 6).alsoToMat(p)(Keep.right).toMat(p2)(Keep.both).run()
+      ps1.request(6)
+      ps2.request(6)
+      ps1.expectNext(1, 2, 3, 4, 5, 6)
+      ps2.expectNext(1, 2, 3, 4, 5, 6)
+      ps1.expectComplete()
+      ps2.expectComplete()
+    }
+
+    "alsoTo must continue if sink cancels" in assertAllStagesStopped {
+      val p, p2 = TestSink.probe[Int](system)
+      val (ps1, ps2) = Source(1 to 6).alsoToMat(p)(Keep.right).toMat(p2)(Keep.both).run()
+      ps2.request(6)
+      ps1.cancel()
+      ps2.expectNext(1, 2, 3, 4, 5, 6)
+      ps2.expectComplete()
+    }
   }
```

Comment:
I wrote this before I saw that the implementation reused the other broadcast. good as is